tests/multithreading/thread_local_storage_06.cc

```diff
@@ -46,7 +46,7 @@ Threads::ThreadLocalStorage<X> tls_data(forty_two);
 
 
 void
-execute(Threads::Mutex &m)
+execute(Threads::Mutex *m)
```

Comment:
same here, and perhaps other tests?

Reply:
I'm using `std::ref` here now and also in other places.